src/main/java/net/atos/zac/zoeken/IndexeerService.java

```diff
@@ -113,14 +113,14 @@ public void indexeerDirect(final List<String> objectIds, final ZoekObjectType ob
     }
 
     @Transactional(Transactional.TxType.NEVER)
-    public Resultaat indexeer(final int batchGrootte, final ZoekObjectType objectType) {
+    public IndexResult indexeer(final int batchGrootte, final ZoekObjectType objectType) {
         if (herindexerenBezig.contains(objectType)) {
             log(objectType, "Indexeren niet gestart, markeren voor herindexeren is nog bezig");
```

Comment:
Do we log in Dutch? I thought that would always be in English.

Reply:
Good call. Will translate it in a future PR.